tracetool/Manifest/libva_trace.man

```diff
@@ -0,0 +1,612 @@
+<?xml version="1.0"?>
```

Comment:
To me man is manual pages. Can you, please, use appropriate extension, i.e. .xml

Reply:
this is manifest to describe libva trace event id and their event data structure.
the manifest format follow window etw manifest, it is xml, but msft define field names and layout to describe event data structure.  msft use .man for window etw manifest, so it is better to follow.
msft have UI tool to edit these manifest, also can edit in txt editor.